internal/namespaces/instance/v1/testdata/test-create-server-simple-gp1xs.stdout.golden

```diff
@@ -0,0 +1,35 @@
+id                        f018721d-0f0b-4229-a0a7-ef00cb91d1e8
+name                      cli-srv-intelligent-euler
+organization              b2593aa3-d0e8-4366-89c3-6e666abe1f6f
+allowed-actions.0         poweron
+allowed-actions.1         backup
+commercial-type           GP1-XS
```

Comment:
we should use `testing.go/TestCheckEqual()` to only test for this value

`.golden` should be used for UI testing 

see https://github.com/scaleway/scaleway-cli/blob/v2/internal/namespaces/instance/v1/instance_cli_test.go#L153